ArcadiaParties-SPA/src/styles.css

```diff
@@ -1 +1,9 @@
-/* You can add global styles to this file, and also import other style files */
+html, body { height: 100%; }
```

Comment:
Could you use code formatting also for styles?

Reply:
Code formatted

Reply:
Code formatted

Reply:
No, it isn't. Formatted code should look like:
```scss
html, body {
  ...
}

body {
  ...
}
```

Reply:
No, it isn't. Formatted code should look like:
```scss
html, body {
  ...
}

body {
  ...
}
```

Reply:
But It already looks like this...

Reply:
Now it looks like this:
```css
body { margin: 0; 
  ...
}
```

Reply:
Code formatted again